src/tle-update.c

```diff
@@ -34,6 +34,9 @@
 #ifdef HAVE_CURL
 #include <curl/curl.h>
 #endif
+#ifdef USE_WIN32_FETCH
+#include "win32-fetch.h"
+#endif
```

Comment:
Sorry for being a pain in the butt, but another comment.
The use of lines such as:
```
#ifdef HAVE_CURL
..
#endif
#ifdef USE_WIN32_FETCH
....
#endif
```
could indicate that *both* combination would be allowed. IMHO, it would better be:
```
#if defined(_WIN32)
...
#elif defined(HAVE_CURL)
...
#endif
```
throughout the code. I mean, what would be the benefit of *not* using `WinInet` functions on Windows?

Reply:
Good point, I added HAVE_CURL in the first place so I could omit curl calls while sorting other dependencies out. That temporary fix can now be removed so we end up with:
```
#ifdef WIN32
    <win32 fetch code>
#else
    <curl code>
#endif
````